tests/test_ddc_overwrite.c

```diff
@@ -0,0 +1,40 @@
+#include <stdint.h>
+#include <stdlib.h>
+
+#include "cheriintrin.h"
+
+/* This test atempts to circumvent the DDC compartmentalization feature by
+ * using the existing PCC as a temporary DDC to access a secret pointer beyond
+ * the original bounds of the compartment. The address of the secret is
+ * expected to be passed as `secret_addr`, and the function will return the
+ * value at that address. If the given `secret_addr` is out of the original DDC
+ * bounds, and this test does not fail, then we have broken the expected
+ * security guarantees of our DDC compartmentalization approach.
+ */
+
+int
+test_overwrite(unsigned long long secret_addr)
```

Comment:
Is "overwrite" correct?

Reply:
Renamed in https://github.com/capablevms/CHERI-ELF-comp/pull/12/commits/42ee892cab7a9700319e993dea5b9f85b78ff504.